components/admin_console/admin_definition.jsx

```diff
@@ -278,6 +278,12 @@ export default {
                                 },
                             ],
                         },
+                        {
```

Comment:
Now this is not needed

Reply:
Oops. Committed that accidentally, thanks.